src/components/Group.js

```diff
@@ -71,8 +72,11 @@ export default function Group(props) {
         {
           edited && <DataMarker />
         }
-        <button class="bio-properties-panel-group-header-button">
-          <GroupArrowIcon class={ open ? 'bio-properties-panel-arrow-down' : 'bio-properties-panel-arrow-right' } />
+        <button
+          title="Toggle section"
```

Comment:
Did we agree to call this _section_ instead of _group_?

Reply:
Not sure whether we ever actively agree on a specific term here, but "section" is part of our [technical writing guide](https://github.com/camunda/camunda-modeler-design/blob/main/docs/technical-writing.md).

"Group" seems to be something we came up technically.